__tests__/hooks/useFeedHeader.test.js

```diff
@@ -54,7 +54,7 @@ describe('Feed Header should run correctly', () => {
     act(() => {
       result.current.onBackNormalUser();
     });
-    expect(mockedNavigate).toHaveBeenCalled();
+    expect(mockedGoBack).toHaveBeenCalled();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `mockedNavigate` to `mockedGoBack` seems appropriate if the functionality being tested has changed from navigation to going back. However, ensure that this change aligns with the intended behavior of the `onBackNormalUser` function.
```diff
-    expect(mockedNavigate).toHaveBeenCalled();
+    expect(mockedGoBack).toHaveBeenCalled();
```
If the function is supposed to navigate to a specific screen rather than just going back, you might need to revert this change or adjust the function implementation accordingly.


<!-- This is an auto-generated comment by OSS CodeRabbit -->